cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/MapReduceRuntimeService.java

```diff
@@ -373,12 +373,32 @@ protected void startUp() throws Exception {
   protected void run() throws Exception {
     MapReduceMetricsWriter metricsWriter = new MapReduceMetricsWriter(job, context);
 
+    int reportInterval = cConf.getInt(Constants.AppFabric.MAPREDUCE_STATUS_REPORT_INTERVAL_SECONDS);
+    String val = context.getRuntimeArguments().get(Constants.AppFabric.MAPREDUCE_STATUS_REPORT_INTERVAL_SECONDS);
+    if (val != null) {
+      try {
+        int interval = Integer.parseInt(val);
+        if (interval < 1) {
+          throw new NumberFormatException("Must be at least 1.");
```

Comment:
The idea is that this gets caught by the surrounding try block. So the message will be similar to the case where parsing the number fails.
```
2017-06-22 11:44:00,196 - WARN  [MapReduceRunner-ComputeSum:c.c.c.i.a.r.b.MapReduceRuntimeService@386] - Invalid value '-1' for 'mapreduce.status.report.interval.seconds' given in runtime arguments: Must be at least 1. Using default of 60 seconds.
```
I will add a comment.